package.json

```diff
@@ -1,58 +1,58 @@
 {
```

Comment:
Please remove this changes. Only the format has been updated

Reply:
I didn't catch you well @markuscodes1. Can you please clarify this change

Reply:
Sure. I think it's not necessary to merge the `package.json` in this PR. There are no functional changes. Only the format has been updated. So I think it's out of scope for this pr.